client/js/components/map/publicdetail/diplankarte/DiplanKarteWrapper.vue

```diff
@@ -157,6 +140,127 @@ const handleDrawing = (event) => {
   emit('locationDrawing', payload)
 }
 
+// Feature: Set location info
+
+const closeLocationInfo = () => {
+  isLocationInfoClosed.value = true
+}
+
+const isLocationInfoClosed = ref(false)
+
+const isLocationToolSelected = computed(() => {
+  return store.state.PublicStatement.activeActionBoxTab === 'draw'
+})
+
+// Feature: Show Overlays/Layers in Layer Switcher and on Map
+
+const buildLayerConfigsList = () => {
+  const layersFromDB = store.getters['Layers/elementListForLayerSidebar'](null, 'overlay', true)
+
+  return layersFromDB
+    .map((layer) => {
+      const layerType = layer.attributes.serviceType?.toLowerCase()
+      const configBuilder = layerConfigBuilders[layerType]
+
+      if (!configBuilder) {
+        console.warn(`No config builder found for layer type: ${layerType}`)
+        return null
```

Comment:
nitpick (there should always be an empty line before a return statement)
```suggestion
        console.warn(`No config builder found for layer type: ${layerType}`)
        
        return null
```

Reply:
changed it in 1cbd271a5ffa9725c50de4bf781da3061619f1f8